### DISCUSSION TITLE
.NET MAUI updates in .NET 9 Preview 6

### DISCUSSION THREAD

**Discussion by davidortinau** [dotnet/maui#23506] [Announcements]
The team is continuing work on core fundamentals of the .NET MAUI SDK to improve overall product quality. This includes expanding test coverage, end to end scenario testing, and bug fixing.

.NET MAUI updates in .NET 9 Preview 6:
* [Release notes](https://github.com/dotnet/core/blob/main/release-notes/9.0/preview/preview6/dotnetmaui.md) 
* [What's new in .NET MAUI in .NET 9](https://learn.microsoft.com/dotnet/maui/whats-new/dotnet-9) documentation.
* [GitHub Release](https://github.com/dotnet/maui/releases/tag/9.0.0-preview.6.24327.7)

.NET 9 Preview 6:
* [Discussion](https://aka.ms/dotnet/9/preview6)
* [Release notes](https://github.com/dotnet/core/blob/main/release-notes/9.0/preview/preview6/README.md)

**Comment by zuo99** [comment]
We are encountering a major issue with MAUI 8 apps. When the APK is handed over to the publisher, the signing process disrupts the alignment. During installation (signature verification) and running the app, we encounter an offset error with the assemblies/assemblies.blob file in the APK.

Since it's impractical to influence the publisher's release process and realign the APK each time, we need a solution that allows the app to run normally without re-aligning and re-signing the APK.
